app/src/main/java/org/akvo/flow/injector/module/ApplicationModule.java

```diff
@@ -106,25 +106,25 @@ public ApplicationModule(FlowApp application) {
 
     @Provides
     @Singleton
-    Context provideContext() {
+    public Context provideContext() {
```

Comment:
the methods of this class were made public which allows to override them by mock injected objects in UI tests